applications/wg-easy/container/Containerfile

```
@@ -0,0 +1,79 @@
+# Base image for all shared Containerfiles for taskfiles
+# Use this image as base image for app specific container files
+FROM --platform=$BUILDPLATFORM ubuntu:24.04
+
+ARG TARGETOS
+ARG TARGETARCH
+
+WORKDIR /tools
+
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive \
+    HOME=/home/devuser \
+    SHELL=/bin/bash
+
+# Install CLI tools
+RUN apt-get update && apt-get install -y \
+    curl \
+    jq \
```

needed for following examples in the README

```suggestion
    jq \
    less \
```